BCP195bis/draft-ietf-uta-rfc7525bis.md

```diff
@@ -407,6 +407,22 @@ Rationale: TLS compression has been subject to security attacks, such as the CRI
 
 Implementers should note that compression at higher protocol levels can allow an active attacker to extract cleartext information from the connection. The BREACH attack is one such case. These issues can only be mitigated outside of TLS and are thus outside the scope of this document. See {{Section 2.6 of RFC7457}} for further details.
       
+### Certificate Compression
+
+Certificate chains often take up the majority of the bytes transmitted during
+the handshake.  In order to manage their size, the following methods can be
+employed:
+
+* Limiting the number of names or extensions;
+* Using keys with small public key representations, like ECDSA; or
+* Using certificate compression.
+
+For TLS 1.3 the `compress_certificate` extension is defined in {{?RFC8879}}.
+For TLS 1.2 the `cached_info` extension is defined in {{?RFC7924}}.
+
+Note that while {{?RFC8879}} compression applies to all handshakes that
+successfully negotiate the extension, {{?RFC7924}} necessitates a successful
```

Comment:
As far as I can see, RFC 7924 is not widely implemented. For example, it is an open issue in OpenSSL. Should we even include it?

Reply:
👍 

I've done a quick survey and it looks like you are right re: support in common stacks.